internal/backends/winit/lib.rs

```diff
@@ -89,6 +89,33 @@ cfg_if::cfg_if! {
     }
 }
 
+fn try_create_window_with_fallback_renderer() -> Option<Rc<dyn WindowAdapter>> {
+    #[cfg(any(
+        feature = "renderer-winit-skia",
+        feature = "renderer-winit-skia-opengl",
+        feature = "renderer-winit-skia-vulkan"
+    ))]
+    {
+        if let Ok(window) = window_factory_fn::<renderer::skia::SkiaRenderer>() {
+            return Some(window);
+        }
+    }
+    #[cfg(any(feature = "renderer-winit-femtovg"))]
+    {
+        if let Ok(window) = window_factory_fn::<renderer::femtovg::GlutinFemtoVGRenderer>() {
+            return Some(window);
+        }
+    }
+    #[cfg(any(feature = "renderer-winit-software"))]
+    {
+        if let Ok(window) = window_factory_fn::<renderer::sw::WinitSoftwareRenderer>() {
+            return Some(window);
+        }
+    }
+
+    return None;
```

Comment:
```suggestion
    None
```

Clippy complains about this kind of things